doc/configuration/Configuration.md

```diff
@@ -0,0 +1,319 @@
+
+```"DataSource": "mmuser:mostest@tcp(dockerhost:3306)/mattermost_test?charset=utf8mb4,utf8"```
+*This is the connection string to the master database. When DriverName set to "postgres" then use a connection string in the form “postgres://mmuser:password@localhost:5432/mattermost_test?sslmode=disable&connect_timeout=10”. This setting can only be changed from config.json file, it cannot be changed from the System Console user interface.*
+
+```"DataSourceReplicas": []```
+*This is a list of connection strings pointing to read replicas of MySQL or PostgreSQL database.  If running a single server, set to DataSource. This setting can only be changed from config.json file, it cannot be changed from the System Console user interface.*
+
+```"MaxIdleConns": 10```
+*Maximum number of idle connections held open to the database.*
+
+```"MaxOpenConns": 10```
+*Maximum number of open connections held open to the database.*
+
+```"Trace": false```
+*"true": Executing SQL statements are written to the log for development.*
+
+```"AtRestEncryptKey": "7rAh6iwQCkV4cA1Gsg3fgGOXJAQ43QVg"```
+*32-character (to be randomly generated via Admin Console) salt available to encrypt and decrypt sensitive fields in database.*
+
+
+### Email Settings
+___
+Settings to configure email signup, notifications, security, and SMTP options. 
+
+### Signup
+
+```"EnableSignUpWithEmail": true```
+*"true": Allow team creation and account signup using email and password; “false”: Email signup is disabled and users are not able to invite new members. This limits signup to single-sign-on services like OAuth or LDAP.*
+
+### Notifications
+
+```"SendEmailNotifications": false```
+*"true": Enables sending of email notifications. Developers may set this field to false to skip email setup for faster development.*
+
+```"RequireEmailVerification": false```
+*"true": Require email verification after account creation prior to allowing login; “false”: Users do not need to verify their email address prior to login. Developers may set this field to false so skip sending verification emails for faster development.*
+XXX: This feature is jenky with 
+
+
+```"FeedbackName": ""```
+*Name displayed on email account used when sending notification emails from Mattermost system.*
+
+```"FeedbackEmail": ""```
+*Email displayed on email account used when sending notification emails from Mattermost system.*
+
+### SMTP
+
+```"SMTPUsername": ""```
+*Obtain this credential from administrator setting up your email server.*
+
+```"SMTPPassword": ""```
+*Obtain this credential from administrator setting up your email server.*
+
+```"SMTPServer": ""```
+*Location of SMTP email server.*
+
+```"SMTPPort": ""```
+*Port of SMTP email server.*
+
+### Security
+
+```"ConnectionSecurity": ""```
+*"none": send email over an unsecure connection; "TLS": communication between Mattermost and your email server is encrypted; “STARTTLS”: attempts to upgrade an existing insecure connection to a secure connection using TLS.*
```

Comment:
I think the sentences need to start with capitals to be consistent with the other descriptions